diagnostic.md

```diff
@@ -22,7 +25,7 @@ at the `OL` DOM node).
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
+<!-- $(".listerItem"). -->
```

Comment:
A few possible ways are `$('li').parent().parent().find('form')`, `$('li').parent().parent().children('form')`, and `$('.listerItem').parent().parent().$('form')`.